### PR TITLE
Bundle eslint and stylelint into a single npm command

### LINKED ISSUE #64: Bundle stylelint & eslint with --fix
I'm super lazy. I want to run one script instead of three 👍

### PR DESCRIPTION
**Pull request by AllegroFox**
Fixes #64

### CODE REVIEW

**Comment by brennie** [comment]
Can you reword your commit message as "Bundle eslint and stylelint into a single npm command"

File: package.json [hunk 1]
```diff
@@ -11,7 +11,10 @@
     "htmlhint": "htmlhint \"src/**/*.html\"",
     "tsc": "tsc",
     "install-hooks": "git config core.hooksPath .hooks",
-    "test": "jest"
+    "test": "jest",
+    "lint:eslint": "npm run eslint -- --fix",
+    "lint:stylelint": "npm run stylelint -- --fix",
+    "lint:all": "npm run lint:eslint && npm run lint:stylelint"
```
Comment (brennie) [inline package.json:17]: Can you rename this to `fixlints:` and rename `lint:all` to `fixlints`

**Review by brennie — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (AllegroFox) [reply package.json:17]: sure 👍

**Comment by AllegroFox** [comment]
> Can you reword your commit message as "Bundle eslint and stylelint into a single npm command"

You mean I shouldn't document my laziness 🤷‍♀️

**Review by brennie — APPROVED** [review]

**Pull request merged**